support/faq.md

```diff
@@ -2,60 +2,223 @@
+
+#### What permissions are required to use I/O Events?  
 
-_**What happens if a webhook is down?**_  
+The various required permissions and entitlements vary according to the events provider,
+(see the list above) some are opened to all Adobe customers, others to enterprise developers or administrators only.
+Some of these events providers will required licensing other will be available to all.
 
-I/O Events will retry up to five times. A webhook is considered inactive after five unsuccessful retries with either no response or a response other than HTTP 2XX. The Journaling API can be used to retrieve events that were published while the webhook is down.
+#### Which subscription types do I/O Events support?  
 
-_**Are there other ways to access I/O Events?**_  
+I/O Events supports [webhooks](../intro/webhooks_intro.md) for near-real time notifications (push) 
+as well as [a Journaling API](../intro/journaling_intro.md) (pull) to grab groups of events at a time.
+
+#### Are there other ways to access I/O Events?
 
 Yes: 
 - [Azuqua](https://www.azuqua.com) provides connectors for both AEM events and Analytics Triggers events.
 - [Microsoft Flow](https://flow.microsoft.com) has a connector for Creative Cloud Asset events.
 
-_**What is JWT and what is it used for?**_  
+## Webhook FAQ
+
+#### What happens if a webhook is down? 
+
+If `Adobe I/O Events` fails to receive a successful response code from your webhook within 10 seconds, 
+it retries the request, including a special header: `x-adobe-retry-count`.
+Adobe I/O Events will retry up to five times using an exponential backoff strategy
+(it will retry up to (1 + 2 + 4 + 8 + 16 =) 31 minutes). 
+After that time, if your webhook is still failing (with either no response or a response other than HTTP 2XX.), 
+Adobe I/O Events will mark it as invalid and stops sending requests.
+
+Note you can then use the Journaling API to retrieve events that were published while your webhook was down.
+and once the webhook is back up, you can re-enable it, see the question below,
+
+#### How can I re-enabled my webhook (disabled after a downtime)? How can I retrieve the events I missed?
+
+To restart the flow of requests, once you have fixed the problem preventing your webhook from responding, 
+you must log into Adobe I/O Developer Console, edit your events registration,
+it will re-trigger a webhook challenge request,and eventually a webhook re-activation.
+
+While your webhook is marked Disabled, Adobe will continue to log events a Journal, 
+you can retrieve all of your events for the past 7 days using [Journaling](../intro/journaling_intro.md).
+
+#### Does every Adobe I/O Events webhook http requests come with a signature? 
+     
+Yes, to allow your webhook to reject forged requests, 
+Adobe I/O Events adds a  [`x-adobe-signature`](../intro/webhooks_intro.md#authenticating-events) 
+header to every single Http request it does to your webhook URL (even the first `challenge` GET request)
```

Comment:
... every single HTTP request it makes to your webhook...

Reply:
done